### PR TITLE
testmap: Run more contexts by default

### PR DESCRIPTION
**Pull request by marusak**
Testing here

### CODE REVIEW

File: lib/testmap.py [hunk 1]
```diff
@@ -139,12 +139,16 @@
         'main': [
             'fedora-33',
             'fedora-33/firefox',
+            'fedora-34',
             'rhel-8-5',
             'rhel-9-0'
         ],
+        'rhel-8-5': [
+            'rhel-8-5',
```
Comment (martinpitt) [inline lib/testmap.py:147]: This ran in #2363 already.

File: lib/testmap.py [hunk 2]
```diff
@@ -139,12 +139,16 @@
...
+            'rhel-8-5/firefox',
```
Comment (martinpitt) [inline lib/testmap.py:148]: This did not run (unknown context), but indeed the chance is very high that it works, too. If not, we'll have to pay the price on the next image refresh :grin:

**Review by martinpitt — APPROVED** [review]
Thanks! Waiting for the final composer test (they are always a bit hard to get green..)

**Comment by marusak** [comment]
> Waiting for the final composer test (they are always a bit hard to get green..)

f-34 is green, f-35 is gonna fail, but I did not move that away from manual as I had hunch it is not just gonna work. Was just trying out my luck here :)

**Comment by marusak** [comment]
@jkozol FYI [fedora-35](https://logs.cockpit-project.org/logs/pull-2364-20210831-160644-6f7b31ae-fedora-35-osbuild-cockpit-composer/log.html) fails on main. I also enabled fedora-34 by default on main.

**Pull request merged**